packages/components/src/InputPhoneNumber/InputPhoneNumber.tsx

```diff
@@ -39,7 +39,11 @@ export function InputPhoneNumber({
 }: InputPhoneNumberProps) {
   const { placeholder, validations, pattern = "(***) ***-****" } = props;
   const errorSubject = placeholder || "Phone number";
-  const getValues = useFormContext();
+  const { getValues } =
+    useFormContext() != undefined
+      ? useFormContext()
+      : // If there isn't a Form Context being provided, get a form for this field.
+        useForm({ mode: "onTouched" });
```

Comment:
I wonder if getting `getValues` from `useForm` is valuable here. I tried it a couple ways in Storybook and it always returned an empty object when using `InputPhoneNumber` without a form. The validation still worked when changed it to:

```ts
const { getValues } = useFormContext();
```

Maybe it's valuable for because it's using the `Controller` component.

But it's just a thought, any ideas around here? Did you find a use case where it was problematic?

Reply:
@MichaelParadis Just tagging you here as you're not the PR creator

Reply:
I was seeing an error without it on in the `InputPhoneNumber` story because it isn't in a `Form` so there isn't a context available